src/signals/incident-management/containers/IncidentDetail/components/ChildIncidents/index.js

```diff
@@ -12,9 +12,10 @@ const Title = styled(Heading)`
 `;
 
 const ChildIncidents = ({ incident }) => {
+  const incidentLinks = incident?._links;
   const children = useMemo(
     () =>
-      incident?._links?.['sia:children']?.map(({ href }) => {
+      incidentLinks?.['sia:children']?.map(({ href }) => {
```

Comment:
What's wrong with the conditional chaining in this case?

Reply:
It looks like prettier, doesn't accept this (2 nullables) when the incident is in the depentency array.

Reply:
I would adjust the linter config so that the code stays the same. What do you think @jpoppe ?

Reply:
@aditudorache I agree with @JanJaap if we will start using GraphQL, conditional chaining of multiple levels is a quite common thing.